presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -16,8 +28,22 @@ make it executable with ``chmod +x``, then run it:
 
 Run the CLI with the ``--help`` option to see the available options.
 
+Authentication
+--------------
+
+You can use your authentication details with the ``--user`` and ``-password``
```

Comment:
This is not just "secured" Presto but is specific to password authentication. Also, `--user` is not required.